.github/workflows/ci.yml

```diff
@@ -140,8 +140,42 @@ jobs:
 
     steps:
 
+      - name: Set fetch depth
+        id: set-fetch-depth
+        env:
+          GITHUB_REPOSITORY: ${{ github.repository }}
+        shell: bash
+        run: |
+          # Default fetch-depth of 1 will only fetch the latest commit
+          # This is fine for non-pull-request events where standard major.minor naming conventions
+          # are used so we do not need to attempt to get the name of the parent branch
+          FETCH_DEPTH=1
+          # For pull-requests, set fetch-depth to one more than the number of commits
+          # so we can later make a best attempt at getting the name of the parent branch
+          if [[ "${{ github.event_name }}" == "pull_request" ]]; then
```

Comment:
By setting this, we are effectively dropping push support for forks (e.g. on creative-commoners) - which is okay, I just want to make sure that's an intentional decision.

Reply:
I'm not sure that's not true, it'll just use a fetch depth of 1.  There's logic elsewhere to extract the minor branch from pulls/x.y/my-fix

Reply:
Ahh fair point, I had forgotten we're explicitly checking for branches that match our naming scheme. The parent branch logic won't even be triggered for those push events.